File: ag.net/ProductLibrary/Product.cs
namespace ProductLibrary
{
    public class Product
    {
        public int ProductId { get; set; }
        public string Title { get; set; }
        public int QuantityAvailable { get; set; }
        public string Description { get; set; }
        public decimal UnitPrice { get; set; }
    }
}

File: agpy/rest-api-sqlite3/models/product.py
from sqlalchemy import Column, Integer, String, Float
from sqlalchemy.ext.declarative import declarative_base

Base = declarative_base()

class Product(Base):
    __tablename__ = 'products'
    ProductID = Column(Integer, primary_key=True, autoincrement=True)
    Title = Column(String, nullable=False)
    Description = Column(String)
    QuantityAvailable = Column(Integer, nullable=False)
    UnitPrice = Column(Float, nullable=False)
